index.html

```diff
@@ -699,6 +699,58 @@ <h2>
     const observer = new ComputePressureObserver(pressureChange);
     observer.observe("cpu");
   </pre>
+  <pre class="example js" title="Usage of takeRecords">
+
+    function pressureChange(entries) {
+      //do something with entries
+    }
+
+    const observer = new ComputePressureObserver(pressureChange);
+    observer.observe("cpu");
+
+    // time-critical section requiring entries
```

Comment:
Maybe instead saying that we are going to disconnect which means that we might loose entries not yet delivered so pull them immediately